boltdesignsystem/bolt
# DS-156 add text link element and doc page template

## Jira

https://pegadigitalit.atlassian.net/browse/DS-156

## Summary

Introduces the first Bolt element named Text Link.

## Details

1. Text Link element is added to packages and PL docs.
2. New doc page template created to keep individual pages consistent. The template mandates a particular information architecture.
3. `bolt-button-native-styles-reset` mixin is created for scenarios where all styles need to be stripped for a `<button>` element.
4. `bolt-text-link` mixin is created for scenarios where a regular text link style should be used.

## How to test

Run the branch locally and read through everything under PL > Elements > Text Link. Make sure all the docs make sense. Create a dummy page and try to use the twig include and HTML, ensure all props work as intended.
